.gitlab-ci.yml

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Should we be adding this file? I don't see it in any of our other supported modules.

Reply:
Removed